provisioning/local/azure.tf

```diff
@@ -0,0 +1,61 @@
+locals {
+  azure_location = "West Europe"
+}
+
+variable "az_tenant_id" {
+  type    = string
+  default = "9b85ee6f-4fb0-4a46-8cb7-4dcc6b262a89" # Keboola
+}
```

Comment:
v čem je to lepší? je to local vývoj a jinde to máme normálně v locals natvrdo

Reply:
nekde je to tak, nekde tak.. ve vysledku je to asi jedno